### PR TITLE
Feature/forget password

### PR DESCRIPTION
**Pull request by Humnaa**
Trello Link : https://trello.com/c/AJobiCqi/83-forget-password

### CODE REVIEW

File: src/containers/Login/Password.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
+import React, { Component } from "react";
+import EmailConfirmation from "./EmailConfirmation";
+import UpdatePassword from "./UpdatePassword";
+
+class Password extends Component {
```
Comment (ayousaf11) [inline src/containers/Login/Password.js]: @Humnaa rename this component and file to "ForgotPassword"

File: src/utilities/index.js [hunk 2]
```diff
@@ -1,4 +1,5 @@
 export const addNotification = (notificationDOMRef, title, type, message) => {
+  debugger
```
Comment (ayousaf11) [inline src/utilities/index.js]: @Humnaa remove this debugger

**Review by ayousaf11 — CHANGES_REQUESTED** [review]

File: src/containers/Login/Password.js (hunk 1 above)
Comment (Humnaa) [reply src/containers/Login/Password.js]: renamed

File: src/utilities/index.js (hunk 2 above)
Comment (Humnaa) [reply src/utilities/index.js]: done

**Review by ayousaf11 — APPROVED** [review]

**Review by ayousaf11 — APPROVED** [review]

**Pull request merged**